dreadnode/cli/platform/constants.py

```diff
@@ -1,12 +1,16 @@
 import typing as t
 
+from dreadnode.constants import DEFAULT_LOCAL_STORAGE_DIR
+
 PlatformService = t.Literal["dreadnode-api", "dreadnode-ui"]
+PLATFORM_SERVICES = t.cast("list[PlatformService]", t.get_args(PlatformService))
 API_SERVICE: PlatformService = "dreadnode-api"
 UI_SERVICE: PlatformService = "dreadnode-ui"
-SERVICES: list[PlatformService] = [API_SERVICE, UI_SERVICE]
-VERSIONS_MANIFEST = "versions.json"
 
 SupportedArchitecture = t.Literal["amd64", "arm64"]
-SUPPORTED_ARCHITECTURES: list[SupportedArchitecture] = ["amd64", "arm64"]
+SUPPORTED_ARCHITECTURES = t.cast("list[SupportedArchitecture]", t.get_args(SupportedArchitecture))
```

Comment:
Little more verbose, but helps prevent deltas in the code as changes to the literal reflect in the list